src/gui/element/settings.rs

```diff
@@ -161,6 +161,67 @@ pub fn data_container<'a, 'b>(
             .push(scale_buttons_row)
     };
 
+    let import_theme_column = {
+        let title_container =
+            Container::new(Text::new(localized_string("import-theme")).size(DEFAULT_FONT_SIZE))
+                .style(style::NormalBackgroundContainer(color_palette));
+
+        let theme_input = TextInput::new(
+            &mut theme_state.input_state,
+            "Paste URL here...",
+            &theme_state.input_url,
+            Interaction::ThemeUrlInput,
+        )
+        .size(DEFAULT_FONT_SIZE)
+        .padding(6)
+        .width(Length::Units(300))
```

Comment:
I think this looks better and should be enough!

```suggestion
        .width(Length::Units(185))
```